t/010-request-id.t

```diff
@@ -0,0 +1,42 @@
+# vim:set ft= ts=4 sw=4 et:
+
+use Test::Nginx::Socket::Lua;
+use Cwd qw(cwd);
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 5);
+
+my $pwd = cwd();
+
+$ENV{TEST_NGINX_HTML_DIR} ||= html_dir();
+
+no_long_string();
+#no_diff();
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: $kong_request_id works
+--- config
+    location /t {
+        content_by_lua_block {
+            local rid = ngx.var.kong_request_id
+            assert(ngx.re.match(rid, "[0-9a-f]{16}"))
```

Comment:
Should the rid a 32 haxadecimal characters string instead of 16?

Reply:
Now we are using 2 uint32_t random number, two numbers cause 8 bytes, so the hex result is 16 bytes.

Reply:
Now it is 16 bytes random bytes ,32 haxadecimal characters string.